docs/semgrep-code/policies.md

```diff
@@ -131,4 +108,45 @@ To disable a rule, follow these steps:
 
 You can also select individual rules under the **Mode** column and disable them one by one.
 
+## Policies filter reference
+
+This section explains columns in the rule pane of the Policies page in detail:
```

Comment:
Possibly intentional, but did you mean to leave out the Labels, Open findings, and Fix rate?

Reply:
Not intentional! Thanks for the catch.

Reply:
Solved by creating a separate reference of filters vs rule entry columns.